modules/oe_translation_poetry/modules/oe_translation_poetry_mock/src/EventSubscriber/ContentTranslationOverviewAlterSubscriber.php

```diff
@@ -135,6 +136,16 @@ public function alterOverview(ContentTranslationOverviewAlterEvent $event): void
           ),
         ];
       }
+      if (isset($translated_jobs[$language->getId()])) {
+        $links['update_translation'] = [
```

Comment:
I would avoid calling this action update. We already have a "translation update" that has nothing to do with this operation. I suggest translation override.

Reply:
I'm ok with calling the link "translation_override" but I think the title needs to be more descriptive about what we are doing. Something like "Receive translation update (mock)" would do it I think?

Reply:
So we have already
* Accept job (mock)
* Translate job (mock)
* Cancel job (mock)
What about having now one of
* Translate job (mock)
* Translate job again (mock)
* Retranslate job (mock)